public/js/util/source-map.js

```diff
@@ -1,29 +1,112 @@
 "use strict";
 
 const { networkRequest } = require("./networkRequest");
-const { SourceMapConsumer } = require("source-map");
+const invariant = require("invariant");
+const { SourceMapConsumer, SourceNode } = require("source-map");
+const { Source } = require("../types");
+const { Task } = require("./task");
+const { getSource } = require("../selectors");
+const first = require("lodash/first");
 
-let sourceMaps = new Map();
+let sourceMapConsumers = new Map();
+let sourceNodes = new Map();
```

Comment:
I had already changed the source map to a consumer map. Will be easy to store the maps in the store